src/commands/skip.ts

```diff
@@ -26,7 +26,7 @@ class SkipCommand implements BaseCommand {
         }
         if (isSkipMajority(message, gameSession)) {
             gameSession.skipAchieved = true;
-            await sendSkipMessage(message, gameSession);
+            sendSkipMessage(message, gameSession);
```

Comment:
Still want consistent ordering of `sendSkipMessage` followed by `sendSongMessage.` We can change the implementation of `sendSkipMessage` to `await` the first promise to retrieve the new message, then call `.delete` on it without `await`

Reply:
Good thinking!